modules/installation-approve-csrs.adoc

```diff
@@ -93,7 +93,12 @@ endif::ibm-z[]
 +
 [NOTE]
 ====
-Because the CSRs rotate automatically, approve your CSRs within an hour of adding the machines to the cluster. If you do not approve them within an hour, the certificates will rotate, and more than two certificates will be present for each node. You must approve all of these certificates. After you approve the initial CSRs, the subsequent node client CSRs are automatically approved by the cluster `kube-controller-manager`. You must implement a method of automatically approving the kubelet serving certificate requests.
+Because the CSRs rotate automatically, approve your CSRs within an hour of adding the machines to the cluster. If you do not approve them within an hour, the certificates will rotate, and more than two certificates will be present for each node. You must approve all of these certificates. After you approve the initial CSRs, the subsequent node client CSRs are automatically approved by the cluster `kube-controller-manager`.
+====
++
+[NOTE]
+====
+For clusters running on platforms that are not machine API enabled, you must implement a method of automatically approving the kubelet serving certificate requests (CSRs). If a request is not approved, then `oc exec`, `oc rsh`, and `oc logs` commands will not succeed, because the serving certificate is required when the API server connects to the kubelet. Any operation that contacts the Kubelet endpoint requires this certificate approval to be in place. The method must watch for new CSRs, confirm that the CSR was submitted by the `node-bootstrapper` service account in the `system:node` or `system:admin` groups, and confirm the identity of the node.
```

Comment:
```suggestion
For clusters running on platforms that are not machine API enabled, such as bare metal, you must implement a method of automatically approving the kubelet serving certificate requests (CSRs). If a request is not approved, then the `oc exec`, `oc rsh`, and `oc logs` commands cannot succeed because the serving certificate is required when the API server connects to the kubelet. Any operation that contacts the Kubelet endpoint requires this certificate approval to be in place. The method must watch for new CSRs, confirm that the CSR was submitted by the `node-bootstrapper` service account in the `system:node` or `system:admin` groups, and confirm the identity of the node.
```

Reply:
Thank you. I have subsequently updated this note and the one in `modules/querying-operator-status-after-installation.adoc`.